frontend/src/components/Tags.jsx

```diff
@@ -0,0 +1,16 @@
+function Tags({ label, type }) {
+  const tagStyles = {
+    condition: 'bg-lavender-200',
+    category: 'bg-[#FFD3D3]',
+    genre: 'bg-[#D3D3F1]',
+  }
+
+  const typeClass = tagStyles[type]
+  return (
+    <span className={`${typeClass} font-sans-sm py-1 px-3 rounded-md`}>
```

Comment:
Please use `clsx` utility to conditionally apply classnames.

```suggestion
    <span className={clsx(...)}>
```